sw/device/tests/BUILD

```diff
@@ -3582,10 +3582,21 @@ opentitan_test(
 opentitan_test(
     name = "rstmgr_alert_info_test",
     srcs = ["rstmgr_alert_info_test.c"],
+    ############
```

Comment:
I think it is better to add comments in this line of code. From the previous comments:

>you need to uncomment from line 3585~3595.
>Once you assign rsa_key in your BUILD, it will override all rsa_key parameters, which I don't want to.
>The key only works for fpga_cw310_sival

So this diagnoses the problem and provides a way to work around it, but it cannot be committed. So the test keeps not working. Is there some better way to handle this so we can get the test to be runnable? The rsa_key setting below cannot be made local to this rule?

Reply:
comment updated. Only other option I can think of is to create separate tests per exec_env, but that is too cumbersome.
As you see this test has been broken since 12/21 (https://github.com/lowRISC/opentitan/blob/fa447d0cdc993726850ef4422a32e473434a6ba0/sw/device/tests/BUILD#L3614-L3620) for all other targets but test_rom.